src/translations/installer_ru.ts

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ru_RU">
+<context>
+    <name>WindowsInstaller</name>
+    <message>
+        <source>__LANGUAGE_NAME__</source>
+        <translatorcomment>This must be the name of one of the NSIS translation files at https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/Contrib/Language%20files/ without the file extension.</translatorcomment>
+        <translation>Русский</translation>
+    </message>
+    <message>
+        <source>BirdtraySectionDescription</source>
+        <translation>Бесплатное уведомление в системном трее о новой почте для Thunderbird.</translation>
+    </message>
+    <message>
+        <source>WinIntegrationSectionName</source>
+        <translation>Интеграция с Windows</translation>
+    </message>
+    <message>
+        <source>WinIntegrationGroupDescription</source>
+        <translation>Выберите, как интегрировать программу в Windows.</translation>
+    </message>
+    <message>
+        <source>AutoRunSectionName</source>
+        <translation>Автозапуск.</translation>
+    </message>
+    <message>
+        <source>AutoRunDescription</source>
+        <translation>Автоматически запускать ${PRODUCT_NAME} после включения.</translation>
+    </message>
+    <message>
+        <source>AutoCheckUpdateSectionName</source>
+        <translation>Автоматическая проверка обновлений.</translation>
```

Comment:
Section name in Windows? The problem ... I don’t know exactly what she’s called in Russian version

Reply:
Sorry, I meant to say that this is a component name too. And because the others don't have a dot at the end, this one shouldn't either:
![Component page](https://user-images.githubusercontent.com/6966049/79865475-82937600-83db-11ea-91df-c902269d3324.png)